src/quo/components/drawers/drawer_top/component_spec.cljs

```diff
@@ -57,7 +57,7 @@
                {:title                "Title"
                 :type                 :account
                 :account-avatar-emoji "🍿"
-                :networks             [{:name :ethereum :short :eth}]
```

Comment:
just a heads up the structure from this sub used here
https://github.com/status-im/status-mobile/pull/17650
will give 
```
[{:network-name :ethereum :short-name "eth"}]
```
one of the reasons for that was not to clash `name` when destructuring with the "name" function from cljs

Reply:
This is a removed line, not added

Reply:
Haha 😅 sorry about that 😆

Reply:
oh but still my comment is relevant. you are using the structure `[:ethereum]`, I am suggesting a different structure ^ ^ 👍